### PR TITLE
Convert FLINT implementation life-cycle to markdown

### PR DESCRIPTION
**Pull request by soumik2012**
Issue no. : #17 
This PR converts the [Google Doc ](https://docs.google.com/document/d/1vNSvKFERxvgUjEpBcG18dGaHhNnBC7hdPkkkcKBkeic/edit#heading=h.2f13emraymka)to Markdown 👍🏻

### CODE REVIEW

**Comment by soumik2012** [comment]
Hello @aornugent Can you please have a look at this?

**Review by aornugent — COMMENTED** [review]
Looking good, thanks @soumik2012

Could you please fix a typo in the file name (plain -> plan) and break the subheadings for each sprint over two lines? ✌️

**Comment by soumik2012** [comment]
@aornugent You can check the PR i have modify the files

File: Planning/generic_plan_in_sprints.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+- Administrative arrangements
+- Identification of pilot area and collect all available data. 
+- Decide on Policy and Reporting Priorities for MRV system (International reporting, results based payments related to forests, National reporting, etc.)
+- Basic training: At least 2 staff will be trained in running the FLINT including installing an example version including data.
+- Review existing Land Cover time series (national and global) + decide on approach for (additional) land-cover maps for the project area. An approach to generate wall-to-wall land cover maps will be developed. Backlog years might be covered by an external party so the in-house team can focus on future maps.
+- Review available data (NFI, reporting, classes, etc.) and select classifiers
+- Review existing landscape, key tree species, past and planned disturbances/management
+- Prioritise the most common existing disturbances (e.g. degradation)
+
+### Sprint 1:  Month 3 to 7: Low accuracy working concept
```
Comment (aornugent) [inline Planning/generic_plan_in_sprints.md:25]: @soumik2012 - sorry I meant break it into two parts on separate lines. For example:
```
## Sprint 1
## Month 3 to 7: Low accuracy working concept
```

Comment (soumik2012) [reply Planning/generic_plan_in_sprints.md]: @aornugent  Done!! 💯

**Comment by soumik2012** [comment]
@aornugent Is that right now ??

**Comment by soumik2012** [comment]
@aornugent Done !! 🧇

File: Planning/generic_plan_in_sprints.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
+# Generic Plan in Sprints to Implement of FLINT-based MRV system 
+
+Below the implementation of a FLINT based MRV system is layed out in phases (aka sprints) of 5 months.
```
Comment (HarshCasper) [inline Planning/generic_plan_in_sprints.md:3]: Replace with `The implementation of a FLINT based MRV system is laid out in phases (aka sprints) of 5 months.`

File: Planning/generic_plan_in_sprints.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+Each cycle the system will produce results and on the basis of these results the next phase (sprint) is planned. Hence the sprints below are only indicative of what could be done and they might be adjusted to the policy needs or findings coming up during the implementation of the earlier sprints. The sprints will be worked out in more detail to develop a budget and identify the necessary human resources.
+
+The steps in the generic plan need to be explained in more detail in a generic implementation template which explains what each bullet below means and the purpose of each step.
+
+The implementation period used below stretches over almost 3 years, i.e. 32 months. Of course an MRV system needs upgrading and adjustments after that period but countries can continue the 5 month cycles of improvement after the first 36 months on their own if necessary with support from the user community around moja global.
+
+### Preparation phase:
```
Comment (HarshCasper) [inline Planning/generic_plan_in_sprints.md:13]: Add a line break

File: Planning/generic_plan_in_sprints.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+### Months 1 and 2: Preparation
+
+- Administrative arrangements
+- Identification of pilot area and collect all available data. 
+- Decide on Policy and Reporting Priorities for MRV system (International reporting, results based payments related to forests, National reporting, etc.)
+- Basic training: At least 2 staff will be trained in running the FLINT including installing an example version including data.
+- Review existing Land Cover time series (national and global) + decide on approach for (additional) land-cover maps for the project area. An approach to generate wall-to-wall land cover maps will be developed. Backlog years might be covered by an external party so the in-house team can focus on future maps.
+- Review available data (NFI, reporting, classes, etc.) and select classifiers
+- Review existing landscape, key tree species, past and planned disturbances/management
+- Prioritise the most common existing disturbances (e.g. degradation)
```
Comment (HarshCasper) [inline Planning/generic_plan_in_sprints.md:23]: Add proper full stops on every pointers

File: Planning/generic_plan_in_sprints.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+### Sprint 1:
```
Comment (HarshCasper) [inline Planning/generic_plan_in_sprints.md:25]: Add a line break after this

File: Planning/generic_plan_in_sprints.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+- Training in land cover / land use time series mapping.
+- Develop forest/non-forest 2010-2020 time-series (possibly using contractor)
+- Approximate growth curves for key forest types using available data
+- Volume to biomass conversion based on estimates
+- Calibrate fire disturbance matrix for different fires based on three indicators: fire types, location and size. (Fire is an example, it will change from country to country depending on local priorities.)
+- Map fire events (including type and size) and add them to a data format that can be read by the FLINT-based system
+- Review computer resources available. Plan implementation in line with available computer capacity.
+- Combine all new data, run system, analyse results, report to policy makers
+- Compare pilot area results with previous reporting for the same areas
+- Uncertainty analysis and resulting suggestions for improvements.
```
Comment (HarshCasper) [inline Planning/generic_plan_in_sprints.md:38]: Add proper full stops on every pointers

File: Planning/generic_plan_in_sprints.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+- Forest Management
+- Forest degradation (caused by disturbances that cannot be detected easily: e.g. firewood collection)
+- Afforestation / Rehabilitation / Restoration
+- Generic insect disturbance
+- Crop Module
+- Livestock and manure module
+- Grasslands Module
+- Wetlands Module
+- Wood products Module
+- Others as required
```
Comment (HarshCasper) [inline Planning/generic_plan_in_sprints.md:138]: Add proper full stops on every pointer

File: Planning/generic_plan_in_sprints.md [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+- Quality control of land cover time series
+- Verification and data collection from the field
+- Develop growth curves for additional forest types and/or agricultural crops + volume to biomass conversion
+- Add and roll-back litter fall, fine roots, decomposition for additional forest types
+- Calibrate wood products module including the collection of the necessary data 
+- Software and database set up
+- Refine projections
+- Combine all new data, run system, analyse results, report to policy makers
+- Compare pilot area results with previous reporting for the same areas
+- Uncertainty analysis and resulting suggestions for improvements
```
Comment (HarshCasper) [inline Planning/generic_plan_in_sprints.md:122]: Add proper full stops on every pointer

File: Planning/generic_plan_in_sprints.md [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+- Develop growth curves for additional forest types + volume to biomass conversion
+- Add and roll-back litter fall, fine roots, decomposition for additional  forest types
+- Calibrate livestock and manure management module 
+- Software and database set up
+- Refine projections
+- Combine all new data, run system, analyse results, report to policy makers
+- Compare pilot area results with previous reporting for the same areas
+- Uncertainty analysis and resulting suggestions for improvements
+
+### Sprint 6:
```
Comment (HarshCasper) [inline Planning/generic_plan_in_sprints.md:107]: Add a line break

File: Planning/generic_plan_in_sprints.md [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+- Quality control of land cover time series
+- Verification and data collection from the field
+- Develop growth curves for additional forest types + volume to biomass conversion
+- Add and roll-back litter fall, fine roots, decomposition for additional  forest types
+- Calibrate livestock and manure management module 
+- Software and database set up
+- Refine projections
+- Combine all new data, run system, analyse results, report to policy makers
+- Compare pilot area results with previous reporting for the same areas
+- Uncertainty analysis and resulting suggestions for improvements
```
Comment (HarshCasper) [inline Planning/generic_plan_in_sprints.md:105]: Add proper full stops on every pointer

File: Planning/generic_plan_in_sprints.md [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+- Develop growth curves for additional forest types or woody crops + volume to biomass conversion 
+- Add and roll-back litter fall, fine roots, decomposition for additional - forest types
+- Calibrate a generic crop module (for annual crops and woody crops) including an agricultural soils module. (or other module depending on national priorities.)
+- Software and database set up
+- Refine projections related to mitigation efforts necessary to meet NDCs.
+- Combine all new data, run system, analyse results, report to policy makers
+- Compare pilot area results with previous reporting for the same areas
+- Uncertainty analysis and resulting suggestions for improvements
+
+### Sprint 5:
```
Comment (HarshCasper) [inline Planning/generic_plan_in_sprints.md:90]: Add a line break

File: Planning/generic_plan_in_sprints.md [hunk 12]
```diff
@@ -0,0 +1,138 @@
...
+- Develop forest/non-forest 2000-2010 time series
+- Develop 1 forest growth curve based the Tier 3 Broadleaf model, alometrics and the related national data + volume to biomass conversion (Can change based on National priorities.)
+- Verification and data collection from the field
+- Parameterise model for litter fall, fine roots, decomposition
+- Calibrate or develop a hurricane disturbance matrix based on the national data for hurricane impact. (Or any other disturbance that is the next highest priority.) 
+- Software and database set up
+- Projection carbon impact of planned mitigation interventions for forest and estimate the effort needed to meet the targets currently included in the NDC.
+- Combine all new data, run system, analyse results, report to policy makers
+- Compare pilot area results with previous reporting for the same area
+- Uncertainty analysis and resulting suggestions for improvements
```
Comment (HarshCasper) [inline Planning/generic_plan_in_sprints.md:54]: Add proper full stops on every pointer

File: Planning/generic_plan_in_sprints.md [hunk 13]
```diff
@@ -0,0 +1,138 @@
...
+- Quality control of land cover time series
+- Verification and data collection from the field
+- Develop growth curves for key forest types + volume to biomass conversion
+- Add and roll-back litter fall, fine roots, decomposition
+- Calibrate a forest degradation disturbance matrix. Possibly a module will be developed to estimate degradation on the basis of proxies. Additional review of literature and national data will determine which approach is most effective. (Or another disturbance that is the next priority.)
+- Software and database set up
+- Refine carbon impact projections of planned interventions
+- Combine all new data, run system, analyse results, report to policy makers
+- Compare pilot area results with previous reporting for the same areas
+- Uncertainty analysis and resulting suggestions for improvements
```
Comment (HarshCasper) [inline Planning/generic_plan_in_sprints.md:71]: Add proper full stops on every pointer

File: Planning/generic_plan_in_sprints.md [hunk 14]
```diff
@@ -0,0 +1,138 @@
...
+
+### Sprint 4:
```
Comment (HarshCasper) [inline Planning/generic_plan_in_sprints.md:73]: Add a line break

File: Planning/generic_plan_in_sprints.md [hunk 15]
```diff
@@ -0,0 +1,138 @@
...
+- Quality control of land cover time series
+- Verification and data collection from the field
+- Develop growth curves for additional forest types or woody crops + volume to biomass conversion 
+- Add and roll-back litter fall, fine roots, decomposition for additional - forest types
+- Calibrate a generic crop module (for annual crops and woody crops) including an agricultural soils module. (or other module depending on national priorities.)
+- Software and database set up
+- Refine projections related to mitigation efforts necessary to meet NDCs.
+- Combine all new data, run system, analyse results, report to policy makers
+- Compare pilot area results with previous reporting for the same areas
+- Uncertainty analysis and resulting suggestions for improvements
```
Comment (HarshCasper) [inline Planning/generic_plan_in_sprints.md:88]: Add proper full stops on every pointer

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Planning/generic_plan_in_sprints.md (hunk 3 above)
Comment (soumik2012) [reply Planning/generic_plan_in_sprints.md]: @HarshCasper done!!

File: Planning/generic_plan_in_sprints.md (hunk 4 above)
Comment (soumik2012) [reply Planning/generic_plan_in_sprints.md]: @HarshCasper Done!!

File: Planning/generic_plan_in_sprints.md (hunk 13 above)
Comment (soumik2012) [reply Planning/generic_plan_in_sprints.md]: @HarshCasper Done!!

File: Planning/generic_plan_in_sprints.md (hunk 15 above)
Comment (soumik2012) [reply Planning/generic_plan_in_sprints.md]: @HarshCasper Done !!

File: Planning/generic_plan_in_sprints.md (hunk 9 above)
Comment (soumik2012) [reply Planning/generic_plan_in_sprints.md]: @HarshCasper Done !!

File: Planning/generic_plan_in_sprints.md (hunk 2 above)
Comment (soumik2012) [reply Planning/generic_plan_in_sprints.md]: @HarshCasper Done!

File: Planning/generic_plan_in_sprints.md (hunk 14 above)
Comment (soumik2012) [reply Planning/generic_plan_in_sprints.md]: Done!!

**Review by HarshCasper — APPROVED** [review]

**Comment by HarshCasper** [comment]
Thanks @soumik2012 for contributing to @moja-global

**Pull request merged**